CpEpEngine.cpp

```diff
@@ -2577,4 +2578,16 @@ STDMETHODIMP CpEpEngine::SignatureVerifies(BSTR text, BSTR signature, VARIANT_BO
     }
     
     return status;
-}
\ No newline at end of file
+}
+
+STDMETHODIMP CpEpEngine::ManagePassphrase(LPSAFEARRAY accounts_with_old_passphrases, BSTR new_passphrase, LPSAFEARRAY* error_accounts) {
```

Comment:
In the confluence documentation the description of this method - managePassphrase - says that it is used to set or unset passphrases for the accounts provided. In case of removing (unset) passphrase for some account how do you manage this through this method? i mean, do we need to pass in all the old passphrases and leave empty the new_passphrase?

Reply:
the idea is to hand in the old passphrase, and the new. there is no concept for multiple old passphrases at the level of an encrypted key

Reply:
Yes, from the RFC I read that an empty new passphrase means no passphrase.